### ISSUE TITLE
Some of the description, effect, item and spell can't find in the game, but it can be seen in "en_us.json"

### ISSUE THREAD

**Issue by Catcatpigeon** [cleannrooster/invocations-spellengine#3]
Also thank you for making such an interesting mod! We are now working on the translaiton of "invocations".
Here are some of our questions: 
1. cant find the spells below in the game
flame stream 
rising flame
scorching ray
freezing aura
deathchill 
glacial hammer
icestorm
hijack
trailblaze

2. in the tooltips of the spell "freezing aura" and "deathchill", there is a same description shown as below
![image](https://github.com/cleannrooster/invocations-spellengine/assets/136223541/deb64a07-c2fa-4c29-acca-22ae99c2aa95)
Since we didn't find the spell in game, we can't test their reall effects. is it means the duration of the 10s freezing will be refresh when a new enemy enter the area, or it's actually talking about the spell will be on cooldown when there is enemy in the area.

3. We saw your description about the spell book"Unbound Invocations" in modrinth, but also can't find it in game, and actually, the only two things we can see in game is "null rune"and “runic invocation”. Besides, like "Arcane Invocations""Fire Invocations""Frost Invocations" and some description like "Invoke arcane power,adding one Invocation Power.", also can't find in game.

**Comment by cleannrooster** [comment]
Yes, due to my own shortcomings in keeping the data files and code tidy
I've left some things that never made it in the actual mod. Feel free to
ignore anything you can't find in the game. Unbound Invocations is no
longer in the mod.

**Comment by Catcatpigeon** [comment]
sure! we got it, thank you for ansewring~^v^
